.travis.yml

```diff
@@ -25,20 +25,23 @@ x_base_steps:
   - &dep_before_install
     before_install:
       - travis_retry make tidy
+      - hack/ci/get-dep.sh
 
   # before_install for jobs that require go builds and do not run for doc-only changes
   - &go_before_install
     before_install:
       # hack/ci/check-doc-only-update.sh needs to be sourced so
       # that it can properly exit the test early with success
       - source hack/ci/check-doc-only-update.sh
+      - hack/ci/get-dep.sh
```

Comment:
Now that we're using go modules for the SDK (as of 0.9.0), I think the dep setup stuff can be removed.

Reply:
took that out